config/lib/addControllers.js

```diff
@@ -45,6 +46,27 @@ module.exports = function (app) {
 			}
 		});
 
+		app.use(function (err, req, res, next) {
+			if (err.shouldReload) {
+				if (!err.status) err.status = 500;
+				res.status(err.status).send(err.message);
```

Comment:
Is `err.message` always defined? Maybe have a fallback `err.message || "Internal server error"`

Reply:
If an error is thrown from mongoose it always contains a message. But you are right in case we want to use the middleware for anything besides mongoose it would be a nice check to have I will add that.